### PR TITLE
3cb factions

### PR DESCRIPTION
**Pull request by Salbei2500**

### CODE REVIEW

File: factions/Peshmerga.hpp [hunk 1]
```diff
@@ -98,37 +96,47 @@ class Peshmerga {
...
+            primaryWeapon = "UK3CB_RPK_74N";
+            primaryWeaponMagazine = "UK3CB_RPK74_60rnd_545x39";
             primaryWeaponOptics = "rhs_acc_1p29";
-            backpack = "SMA_BackPack_Tan";
             addItemsToBackpack[] = {
-                LIST_3("rhs_100Rnd_762x54mmR_7N26"),
-                "rhs_acc_1p29"
+                LIST_3("UK3CB_RPK74_60rnd_545x39")
             };
         };
```
Comment (McDiod) [inline factions/Peshmerga.hpp]: missing `backpack` here

File: factions/Peshmerga.hpp [hunk 2]
```diff
@@ -98,37 +96,47 @@ class Peshmerga {
...
+                    GRAD_FACTIONS_MEDICITEMS_PT
+                };
+                class LIEUTENANT: SERGEANT {};
+                class CAPTAIN: SERGEANT {};
+                class MAJOR: SERGEANT {};
+                class COLONEL: SERGEANT {};
+            };
         };
 
         //Explosive Specialist
```
Comment (McDiod) [inline factions/Peshmerga.hpp:140]: missing `backpack` here

File: factions/Taliban.hpp [hunk 3]
```diff
@@ -78,30 +122,64 @@ class Taliban {
...
         class soldier_AAA_F: Soldier_F {
-            uniform = "LOP_U_AM_Fatigue_03_6";
-            backpack = "B_Carryall_khk";
+            vest = "";
+            backpack[] = {	
+                "UK3CB_B_Alice_K",
+				"B_Carryall_khk",
+				"B_Carryall_cbr",			
+				"UK3CB_CHC_C_B_HIKER",
+				"rhs_rpg_empty"
```
Comment (McDiod) [inline factions/Taliban.hpp:131]: Do these hold 3 Igla rockets?

**Review by McDiod — APPROVED** [review]
https://github.com/gruppe-adler/grad-factions/pull/38#discussion_r610600702

I take it they do.

**Review by nomisum — APPROVED** [review]

**Review by DerZade — APPROVED** [review]

**Comment by nomisum** [comment]
@AdlerSalbei du hast die Ehre des merge

**Pull request merged**